youtrack-backup.pl

```diff
@@ -94,12 +89,12 @@ ()
 $curl_query2 = "--data '{ \"backupStatus\": { \"backupInProgress\": true, \"stopBackup\": false } }'";
 $curl_cmd    = "curl $auth_header $content_header $progress_header $curl_query1 $curl_query2";
 
-printf "%s: Backing up YouTrack database\n", tstamp() if (! $quiet);
+output_if_not_quiet("Backing up YouTrack database\n");
 $result = `$curl_cmd | jq '.backupStatus.backupError'`;
 
-chomp($result);
-if ($result ne "null") {
-	die "Database backup failed: $result\n";
+chomp $result;
+if ( $result ne 'null' ) {
+    croak "Database backup failed: $result\n";
```

Comment:
I think this one should still be `die`. See https://www.perlmonks.org/?node_id=685452

> You use die when the error is something you or your code didn't do right. You use croak when it's something your caller isn't doing right.